packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -515,6 +516,8 @@ class _CupertinoContextMenuState extends State<CupertinoContextMenu> with Ticker
       _childHidden = true;
     });
 
+    HapticFeedback.mediumImpact();
```

Comment:
Can you provide a reasoning for this? Is this something native iOS does?

Reply:
In my new changes I moved it to another section. But yes, on iOS there is a native haptic feedback when "committing" to opening the menu (~350ms)